README.md

```diff
@@ -31,40 +31,45 @@ Demonstrate your understanding of this week's concepts by answering the followin
 Edit this document to include your answers after each question. Make sure to leave a blank line above and below your answer so it is clear and easy to read by your project manager
 
 1. What is the DOM?
+   It is called the document object model and it is an object that contains everything on the page
 
 2. What is an event?
+   An event is a huge object responsible for a bunch of things occuring on the page, example: MouseEevent, and it refers to the initial Event object
```

Comment:
Events are actions that can be detected by JavaScript, and the event object gives information about the event that has occurred.